pkg/component/load.go

```diff
@@ -5,9 +5,154 @@
 package component
 
 import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
 	"github.com/elastic/go-ucfg/yaml"
 )
 
+const specGlobPattern = "*.spec.yml"
+
+var (
```

Comment:
This errors seems only related to input should they be prefixed by "input is not supported" ?

Reply:
Yes, will fix.